packages/design-system/src/Countdown.stories.tsx

```diff
@@ -1,22 +1,22 @@
-import React, { useMemo, useState } from 'react'
+import { useMemo, useState } from 'react'
 import Countdown from './Countdown'
 
 export const Default = () => {
-  let time = new Date()
+  const time = new Date()
```

Comment:
Even though it should work technically, does it make sense to declare a constant time, which is then changed in the following lines using the `setSeconds` method? Simply out of interest: What is the best practice in such cases - to declare a variable to be constant whenever possible or when it remains semantically constant?